src/comodo/mujocoSimulator/mujocoSimulator.py

```diff
@@ -34,41 +50,90 @@ def set_visualize_robot_flag(self, visualize_robot):
 
     def set_base_pose_in_mujoco(self, xyz_rpy):
         base_xyz_quat = np.zeros(7)
-        base_xyz_quat[:3] = xyz_rpy[:3]
+        base_xyz_quat[:3] = xyz_rpy[:3] + 0.0001
```

Comment:
This looks weird. Are you setting a quaternion from an rpy? Also what is the 0.0001 for?

Reply:
`base_xyz_quat` contains the base XYZ position and the orientation as  quaternion, in the order xyz_quat
meanwhile `xyz_rpy` contains the base `xyz` position and orientation as `rpy` in the order xyz_rpy. 
Therefore I am setting the XYZ position of the base, adding  `0.0001` to the z component to lift from the ground (even if the value is un-appreciable )

Reply:
Ah I understand. Still the 0.0001 is a bit of a magic number. If it is needed to make things working, then I would suggest to promote it as a parameter.